usaspending_api/search/v2/views/spending_by_award.py

```diff
@@ -57,7 +57,7 @@
 from usaspending_api.common.exceptions import UnprocessableEntityException
 from usaspending_api.submissions.models import SubmissionAttributes
 
-logger = logging.getLogger("console")
+logger = logging.getLogger(__name__)
```

Comment:
I expected a lot more could be removed from this file. If this endpoint is completely done using materialized views what is `matview_search_filter_determine_award_matview_model()` used for? Same with all of the non-elasticsearch/non-subaward mappings

Reply:
Will need to step through the imports some more.